lmdeploy/pytorch/kernels/pagedattention.py

```diff
@@ -201,25 +346,25 @@ def _reduce_split_kernel(
     stride_obs,
     stride_oh,
     stride_od,
-    head_size: tl.constexpr,
+    head_size_v: tl.constexpr,
     SPLIT_K: tl.constexpr,
-    BLOCK_DMODEL: tl.constexpr,
+    BLOCK_DV: tl.constexpr,
```

Comment:
what does 'DV' refer to?

Reply:
head_**D**im of **V**alue.

value could have a different head_dim from key.